docs/developer/CI.md

```diff
@@ -0,0 +1,410 @@
+    [manual parametrised runs](#manual-parametrised-runs)) is not an `opt=debug`
+    build, you may miss most warnings.
+
+The reports allow the types of issues to be selected and filtered:
+
+- They can be grouped and sorted by selecting the appropriate buttons above the
+  main table and table headers.
+
+- The doughnut of warning severity is actually a clickable filter to filter to
+  warnings of that level!
+
+- The `+` icon to the left of the issue in details mode allows you to expand the
+  issue to get more detail.
+
+!!! failure "Known issue"
+
+    Due to the relatively large sizes of CASTEP source files, while selecting the
+    `file:line` link will take you to an annotated copy of the source at the point
+    of the issue in question, the rendering engine tends to bog down machines
+    causing browsers to struggle or fail to load the annotated file.
+
+##### FORD API Documentation
+
+As part of the full build of the `default` branch following a merge, Jenkins
+will trigger a [`FORD`](https://forddocs.readthedocs.io/en/latest/) API
+documentation build. The results of this build are accessible from the
+`docs-trigger` pipeline in the side-bar.
+
+##### Test Coverage Report
+
+Whenever a `coverage` build is run (i.e. with either the `ifort` or `gfortran`
+compiler and `opt=coverage`), Jenkins will accumulate the coverage reports and
+make them accessible in the side-bar.
+
+!!! failure "Known issue"
+
+    Due to current security settings on ANVIL the `ifort` reports are not
+    viewable through this view and must be downloaded to be viewed.
+
+### Default builds run
+
+In general, the default set of configurations are run on `rocky8` using:
+
+- Intel 23
+- Intel 21
+- Ifx 24
+- GCC 13
+- GCC 9
+- NAG 7.1
+- NVFortran 23.7
+
+with different combinations of `arch`, `comms` and external libraries attempting
+to cover a broad scope of possible use-cases.
+
+At time of writing, these keys are:
+
+```
+rocky8;linux_x86_64_ifort;debug;mpi#intel23
+rocky8;linux_x86_64_gfortran;debug;mpi#gcc9
+rocky8;linux_x86_64_nag;debug;serial
+rocky8;linux_x86_64_gfortran10;coverage;mpi#gcc13-libxc
+rocky8;linux_x86_64_ifort;coverage;serial#intel23-nocmake
+rocky8;linux_x86_64_ifort;fast;serial#intel23-nocmake
+rocky8;linux_x86_64_gfortran;fast;mpi#gcc9-nocmake
+rocky8;linux_x86_64_gfortran10;fast;mpi#gcc13-nocmake
+rocky8;linux_x86_64_gfortran10;fast;serial#gcc13-nocmake
+rocky8;linux_x86_64_nvfortran;fast;mpi#nogrimmed4
+rocky8;linux_x86_64_ifx;fast;mpi#ifx24-nocmake
+```
+However, these are subject to change.
+
+!!! note
+
+    All `default` or manual builds will also run a test to ensure that the `dist`
+    target of the CASTEP build system matches the files expected relative to the
+    `git` information. This is to ensure that any new files which have been added to
+    the `git` tracking which are not accounted for in `dist` will be detected until
+    accounted for or explicitly excluded.
```

Comment:
Unnecessarily over-wordy.  All it needs to say is
"Default" and "manual" builds also include a test of the correctness and completeness of the source distribution GNU make and CMake "dist" targets.